src/components/Nav.jsx

```diff
@@ -23,49 +19,45 @@ export default function Nav({ filter, setFilter, setReviews,reviews }) {
         (review) => review.category === filter
       );
       setReviews(filteredReviews);
-   
-    } 
+    }
   }, [filter]);
 
   function resetReviews() {
     setIsLoading(true);
-    getReviews()
-      
-      .then((reviewsFromApi) => {
-        setReviews(reviewsFromApi);
-        setIsLoading(false);
-      })
-      getCategories().then((categoriesFromApi) => {
-        setCategories(categoriesFromApi);
-        navigate("/");
-        setIsLoading(false);
-      });
+    getReviews().then((reviewsFromApi) => {
+      setReviews(reviewsFromApi);
+      setIsLoading(false);
+    });
+    getCategories().then((categoriesFromApi) => {
+      setCategories(categoriesFromApi);
+      navigate("/");
+      setIsLoading(false);
+    });
   }
-  const navigate = useNavigate()
+
+  const navigate = useNavigate();
   function handleDropDown(e) {
```

Comment:
Also the drop down box is still present even on individual reviews, which is a bit confusing because there is nothing to sort on that page. Maybe consider moving it out of your nav.